qanary-component-QB-DateOfDeathDBpedia/src/main/java/eu/wdaqua/qanary/component/dateofdeathrealpersons/qb/controller/DefaultController.java

```diff
@@ -0,0 +1,28 @@
+package eu.wdaqua.qanary.component.dateofdeathrealpersons.qb.controller;
+
+import eu.wdaqua.qanary.component.dateofdeathrealpersons.qb.QueryBuilderDateOfDeathDBpedia;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+
+@Controller
+public class DefaultController {
+
+    @Autowired
+    private QueryBuilderDateOfDeathDBpedia queryBuilderDateOfDeathDBpedia;
+
+    @GetMapping("/getdbpediaquery")
```

Comment:
@dschiese Please add an OpenAPI annotation here to make it visible on the swagger-ui.html page.

Reply:
Done here: https://github.com/WDAqua/Qanary-question-answering-components/commit/e37fbf843c9fb8896f1ccdab6583270d36fd91cb